hdl/syn/bit_to_nsvf.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+if [ ! $# -eq 1 ]
+then
+  echo "Wrong usage! $0 <SYN_PROJECT>"
+  exit
+fi
+
+SYN_PROJECT=$1
+AUX_FILES_PATH=../../..
+SVF_TO_NSVF_BIN=svf_to_nsvf-linux-x86.bin
+
+cd ${SYN_PROJECT}.runs/impl_1/
+
+awk -v var=$SYN_PROJECT \
+  '{gsub(/SYN_PROJECT/,var); gsub(/SVF_NAME/,var"_with_fmc"); print}' \
+  ${AUX_FILES_PATH}/bit_to_mcs_to_svf.cmd > \
+    bit_to_mcs_to_svf_${SYN_PROJECT}_with_fmc.cmd
+
+awk -v var=$SYN_PROJECT \
+  '{gsub(/SYN_PROJECT/,var); gsub(/SVF_NAME/,var"_without_fmc"); \
+  gsub(/create_hw_device -part xc2c256_1/,""); print}' \
+  ${AUX_FILES_PATH}/bit_to_mcs_to_svf.cmd > \
+    bit_to_mcs_to_svf_${SYN_PROJECT}_without_fmc.cmd
```

Comment:
It would be way cleaner if you use environmental variables to indicate if the the project name. You can read environmental variables from the tcl side using `$::env(VAR_NAME)`. Also, it would be better to generate both SVF/NSVF with and without the +4SFP-FMC in a single tcl script, no need to split it in two steps.

Remember that tcl is a general purpose language, you can do much more with it than with shell scripting. Actually, you could completely remove the shell script, write a single tcl script and set the `STEPS.WRITE_BITSTREAM.TCL.POST` hook to call it automatically (https://docs.xilinx.com/r/en-US/ug894-vivado-tcl-scripting/Defining-Tcl-Hook-Scripts).

Hint: when dealing with simple text manipulation, prefer using `sed` than `awk`, it has a much simpler syntax.